From 63edb0dcbeec16f00d5a9e54e80e1a0a5b2d6488 Mon Sep 17 00:00:00 2001
From: Marek Kasik <mkasik@redhat.com>
Date: Mon, 20 Jun 2011 15:02:39 +0000
Subject: Printers: Include cups/ppd.h explicitly

cups/ppd.h is not included by cups/cups.h since CUPS 1.5.
---
Index: gnome-control-center-3.0.2/configure.ac
===================================================================
--- gnome-control-center-3.0.2.orig/configure.ac	2011-05-24 10:49:08.000000000 +0200
+++ gnome-control-center-3.0.2/configure.ac	2011-10-14 14:00:39.578591281 +0200
@@ -145,7 +145,7 @@
   CUPS_API_MAJOR=`echo $ECHO_N $CUPS_API_VERSION | cut -d . -f 1`
   CUPS_API_MINOR=`echo $ECHO_N $CUPS_API_VERSION | cut -d . -f 2`
 
-  AC_CHECK_HEADERS([cups/cups.h cups/http.h cups/ipp.h],,
+  AC_CHECK_HEADERS([cups/cups.h cups/http.h cups/ipp.h cups/ppd.h],,
                    AC_MSG_ERROR([CUPS headers not found but CUPS support requested]))
 
   if ! test $CUPS_API_MAJOR -gt 1 -o \
Index: gnome-control-center-3.0.2/panels/printers/pp-new-printer-dialog.c
===================================================================
--- gnome-control-center-3.0.2.orig/panels/printers/pp-new-printer-dialog.c	2011-05-09 12:31:40.000000000 +0200
+++ gnome-control-center-3.0.2/panels/printers/pp-new-printer-dialog.c	2011-10-14 14:00:39.582591231 +0200
@@ -31,6 +31,7 @@
 #include <gtk/gtk.h>
 
 #include <cups/cups.h>
+#include <cups/ppd.h>
 
 #include "pp-new-printer-dialog.h"
 #include "pp-utils.h"
Index: gnome-control-center-3.0.2/panels/printers/pp-utils.c
===================================================================
--- gnome-control-center-3.0.2.orig/panels/printers/pp-utils.c	2011-05-09 12:31:40.000000000 +0200
+++ gnome-control-center-3.0.2/panels/printers/pp-utils.c	2011-10-14 14:00:39.582591231 +0200
@@ -25,6 +25,7 @@
 #include <glib/gstdio.h>
 #include <gtk/gtk.h>
 #include <cups/cups.h>
+#include <cups/ppd.h>
 #include <dbus/dbus-glib.h>
 
 #include "pp-utils.h"
